pages/1.10/installing/oss/cloud/gce/index.md

```diff
@@ -6,283 +6,241 @@ navigationTitle: GCE
+expiration = "6h"
+dcos_security = "permissive"
+dcos_cluster_docker_credentials_enabled =  "true"
+dcos_cluster_docker_credentials_write_to_etc = "true"
+dcos_cluster_docker_credentials_dcos_owned = "false"
+dcos_cluster_docker_registry_url = "https://index.docker.io"
+dcos_overlay_network = <<EOF
+# YAML
+    vtep_subnet: 44.128.0.0/20
+    vtep_mac_oui: 70:B3:D5:00:00:00
+    overlays:
+      - name: dcos
+        subnet: 12.0.0.0/8
+        prefix: 26
+EOF
+dcos_rexray_config = <<EOF
+# YAML
+  rexray:
+    loglevel: warn
+    modules:
+      default-admin:
+        host: tcp://127.0.0.1:61003
+    storageDrivers:
+    - ec2
+    volume:
+      unmount:
+        ignoreusedcount: true
+EOF
+dcos_cluster_docker_credentials = <<EOF
+# YAML
+  auths:
+    'https://index.docker.io/v1/':
+      auth: Ze9ja2VyY3licmljSmVFOEJrcTY2eTV1WHhnSkVuVndjVEE=
+EOF
+gcp_ssh_pub_key_file = "INSERT_PUBLIC_KEY_PATH_HERE"
+```
+_Note: The YAML comment is required for the DC/OS specific YAML settings._
 
-1.  Prefix your username, followed by a colon, to the above line. Also
+## Upgrading DC/OS
 
-    ```bash
-    ajazam:ssh-rsa abcdefghijklmaasnsknsdjfsdfjs;dfj;sdflkjsd ajazam
-    ```
+You can upgrade your DC/OS cluster with a single command. This terraform script was built to perform installs and upgrade from the inception of this project. With the upgrade procedures below, you can also have finer control on how masters or agents upgrade at a given time. This will give you the ability to change the parallelism of master or agent upgrades.
 
-1.  Save contents of `id_rsa.pub`.
+### DC/OS Upgrades
 
-1.  Add the rsa public key to your project
+#### Rolling Upgrade
+###### Supported upgraded by dcos.io
 
-    ```bash
-    chmod 400 ~/.ssh/id_rsa
-    gcloud compute project-info add-metadata --metadata-from-file sshKeys=~/.ssh/id_rsa.pub
-    ```
+##### Masters Sequentially, Agents Parellel:
+```bash
+terraform apply -var-file desired_cluster_profile.tfvars -var state=upgrade -target null_resource.bootstrap -target null_resource.master -parallelism=1
+terraform apply -var-file desired_cluster_profile.tfvars -var state=upgrade
+```
 
-### Configure Docker
+##### All Roles Simultaniously
+###### Not supported by dcos.io but it works without dcos_skip_checks enabled.
 
-1.  You must disable SELinux for Docker to work. Make the following change to `/etc/selinux/config`:
+```bash
+terraform apply -var-file desired_cluster_profile.tfvars -var state=upgrade
+```
 
-    ```bash
-    SELINUX=disabled
-    ```
 
-1.  Reboot host.
+## Maintenance
 
-1.  To install Docker add the Yum repo.
+If you would like to add more or remove (private) agents or public agents from your cluster, you can do so by telling terraform your desired state and it will make sure it gets you there. For example, if I have 2 private agents and 1 public agent in my `-var-file` I can always override that flag by specifying the `-var` flag. It has higher priority than the `-var-file`.
```

Comment:
In this sentence:  "If you would like to add more or remove (private) agents or public agents from your cluster.."
remove the word "more" (as it's superfluous).